ZupIT/ritchie-cli
# Functional tests rit metrics

### Description

Functional tests for rit metrics command
### How to verify it
Check the workflows executed in the Actions menu or make changes to the metrics.go and metrics_test.go files to validate the results.
### Changelog

Validates 3 scenarios.
With 'yes' value, with 'no' value and invalid value.
---------------------------------------------### This pull request generated the following artifacts.
To test the health and quality of this implementation, download the respective binary for your operating system, unzip and directly run the binary like the examples below.
- **Windows** Download the file: **[rit-windows.zip](https://github.com/ZupIT/ritchie-cli/suites/3171641318/artifacts/73058649)** Unzip to some folder like: `C:\home\user\downloads\pr1002` Access the folder: `cd C:\home\user\downloads\pr1002` Directly call the binary: `.\rit.exe --version` or `.\rit.exe name of formula`  - **Linux** Download the file: **[rit-linux.zip](https://github.com/ZupIT/ritchie-cli/suites/3171641318/artifacts/73058647)** Unzip to some folder like: `/home/user/downloads/pr1002` Access the folder: `cd /home/user/downloads/pr1002` Assign execute permission to binary: `chmod +x ./rit` Directly call the binary: `./rit --version` or `./rit name of formula`  - **MacOS** Download the file: **[rit-macos.zip](https://github.com/ZupIT/ritchie-cli/suites/3171641318/artifacts/73058648)** Unzip to some folder like: `/home/user/downloads/pr1002` Access the folder: `cd /home/user/downloads/pr1002` Assign execute permission to binary: `chmod +x ./rit` Directly call the binary: `./rit --version` or `./rit name of formula`> Generated at Tue Jul 06 2021 17:49:11 GMT+0000 (Coordinated Universal Time)
